content/docs/k8s/_index.md

```diff
@@ -0,0 +1,4 @@
++++
```

Comment:
Let's keep these docs in the "getting-started" section, since they're specifically about installing Kubeflow. (I'm hoping we won't get a set of docs per solution, but just one setup guide per solution. Let me know if that's not a realistic hope.)

To move the docs into the getting-started section:

* Move this doc from `content/docs/k8s/_index.md` to `content/docs/started/k8s/_index.md
* Give this doc a weight that puts it near the top of the "getting started" section. To do that, insert a new line between lines 3 and 4, with the following content: `weight = 1`. (We may need to juggle the weights of the other pages if this doesn't work as expected. Basically, the lowest weight goes at the top of the tree within a given section.)
* Move `content/docs/started/getting-started-k8s.md ` to `content/docs/started/k8s/overview.md`.
* Move the other 2 pages to `content/docs/started/k8s/kfctl-existing-arrikto.md` and `content/docs/started/k8s/kfctl-k8s-istio.md`.